src/Symfony/Bundle/Resources/config/doctrine_mongodb_odm.xml

```diff
@@ -22,8 +22,9 @@
             <tag name="api_platform.data_persister" priority="-1000" />
         </service>
 
-        <service id="api_platform.doctrine_mongodb.odm.state.processor" class="ApiPlatform\Doctrine\Orm\State\Processor">
+        <service id="api_platform.doctrine_mongodb.odm.state.processor" class="ApiPlatform\Doctrine\Common\State\Processor">
```

Comment:
Service name is not related to MongoDB anymore, maybe rename the service and create an alias?

Reply:
It is because we need to inject the right Doctrine manager registry (MongoDB ODM and ORM don't have the same one).